config/mfDefs.gem

```diff
@@ -34,11 +34,14 @@ UserStaticLinkFlags  =
 UserDynamicLinkFlags =
 
 IncludeDirs+=$(XDAQ_ROOT)/include
+IncludeDirs+=/usr/local/include/python2.7
 LibraryDirs+=$(XDAQ_ROOT)/lib
 DependentLibraryDirs+=$(XDAQ_ROOT)/lib
+DependentLibraryDirs+=/usr/local/lib
 
 DependentLibraries+=asyncresolv boost_system config log4cplus
 DependentLibraries+=numa toolbox uuid xcept xdaq2rc xerces-c
+DependentLibraries+=python2.7
 
```

Comment:
OK, I see what you mean... we definitely need to fix it and make sure it's done correctly, but for now I guess we leave it.
I would think that it should only be necessary in gemutils, but maybe something's not set up properly